packages/parser/tests/unit/schema/lambda.test.ts

```diff
@@ -1,19 +1,37 @@
 import { describe, expect, it } from 'vitest';
 import { LambdaFunctionUrlSchema } from '../../../src/schemas/';
-import { TestEvents } from './utils.js';
+import { getTestEvent } from './utils.js';
 
-describe('Lambda ', () => {
-  it('should parse lambda event', () => {
-    const lambdaFunctionUrlEvent = TestEvents.lambdaFunctionUrlEvent;
+describe('Schema: LambdaFunctionUrl ', () => {
+  const eventsPath = 'lambda';
 
-    expect(LambdaFunctionUrlSchema.parse(lambdaFunctionUrlEvent)).toEqual(
-      lambdaFunctionUrlEvent
-    );
+  it('should throw when the event is invalid', () => {
+    const event = getTestEvent({ eventsPath, filename: 'invalid' });
+
+    expect(() => LambdaFunctionUrlSchema.parse(event)).toThrow();
+  });
+
+  it('parses a valid event', () => {
+    const event = getTestEvent({ eventsPath, filename: 'get-request' });
+
+    const parsedEvent = LambdaFunctionUrlSchema.parse(event);
+
+    expect(parsedEvent).toEqual(event);
   });
 
-  it('should parse url IAM event', () => {
-    const urlIAMEvent = TestEvents.lambdaFunctionUrlIAMEvent;
+  it('parses event with trailing slash', () => {
+    const event = getTestEvent({ eventsPath, filename: 'with-trailing-slash' });
+
+    const parsedEvent = LambdaFunctionUrlSchema.parse(event);
+
+    expect(parsedEvent).toEqual(event);
+  });
+
+  it('parses iam event', () => {
+    const event = getTestEvent({ eventsPath, filename: 'iam-auth' });
```

Comment:
If possible, let's add the `// Prepare`, `// Act`, `// Assess` comments - it makes the test much easier to scan at a glance.